api/migrations/schema/Version20240609164231.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240609164231 extends AbstractMigration {
+    public function getDescription(): string {
+        return 'Rename camp name to shortname and switch shortname with title if shortname is longer than 16 characters';
+    }
+
+    public function up(Schema $schema): void {
+        $this->addSql('ALTER TABLE camp RENAME COLUMN name TO shortname');
+        $this->addSql('UPDATE camp SET shortname = title, title = shortname WHERE char_length(shortname) > 16 AND char_length(title) <= 16;');
+        $this->addSql('ALTER TABLE camp ALTER title DROP NOT NULL');
+    }
+
+    public function down(Schema $schema): void {
+        $this->addSql('ALTER TABLE camp ALTER title SET NOT NULL');
+        $this->addSql('UPDATE camp SET shortname = title, title = shortname WHERE char_length(shortname) > 16 AND char_length(title) <= 16;');
```

Comment:
This statement doesn't revert the same statement from the up-portion. Actually, there's no easy way to revert this migration without an additional marker of which columns have been switched. So I think we could also just remove this statement, so the down-operation would revert the schema changes but not the actual data migration.